index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello world!</h1>
```

Comment:
Have added just this single line--very much just as a test and a studying repetition